Makefile

```diff
@@ -57,8 +57,8 @@ mock_gen:
 
 .PHONY: test
 test:
-	go test -race ./...
-	cd konnectivity-client && go test -race ./...
+	go test -race -covermode=atomic -coverprofile=server.out ./... && go tool cover -html=server.out -o=server.html
```

Comment:
the name `server` is misleading because we use the top level module with both server and agent. Maybe a name like `test-coverage`, or `konnectivity`?